specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/preview/2015-06-15/Network.json

```diff
@@ -75,6 +75,98 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Network.Admin/locations": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the list of supported locations.": {
+            "$ref": "./examples/Operations/List.json"
+          }
+        },
+        "description": "Returns the list of supported locations",
+        "tags": [
+          "Network"
+        ],
+        "operationId": "Locations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationsList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Network.Admin/locations/{location}/operationResults": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the list of operation results for a location": {
+            "$ref": "./examples/Locations/List.json"
+          }
+        },
+        "description": "Returns the list of operation results for a location",
+        "tags": [
+          "Network"
+        ],
+        "operationId": "LocationsOperationResults_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter",
+            "$ref": "#/parameters/LocationParameter"
```

Comment:
I think lint report error on this line. You seem to have 2 ref in same object. I guess it should be in 2 objects.

Reply:
Thank you! I updated the PR with my new changes and added a more meaningful title to the commit.